commit 4359f319666feca832b5812d83b3f102c5cc0b06
Author: Alex Wilson <alex.wilson@joyent.com>
Date:   2019-06-07T22:01:27+00:00 (4 months ago)
    
    MANTA-4328 zkstream missed wakeups lead to binder sadness
    Reviewed by: Robert Mustacchi <rm@joyent.com>
    Approved by: Robert Mustacchi <rm@joyent.com>

diff --git a/package.json b/package.json
index 943a519..147a359 100644
--- a/package.json
+++ b/package.json
@@ -22,7 +22,7 @@
     "vasync": "2.1.0",
     "verror": "^1.10.0",
     "xtend": "1.0.3",
-    "zkstream": "0.11.3"
+    "zkstream": "0.11.5"
   },
   "engines": {
     "node": ">=0.10"
